src/bootstrap/bootstrap_frontend.sh

```diff
@@ -82,4 +82,4 @@ cd ../../
 rm start_fact_frontend
 ln -s src/start_fact_frontend.py start_fact_frontend
 
-exit 0
\ No newline at end of file
+exit 0
```

Comment:
Error when starting with auth enabled (after running the boostrap script):
passlib.exc.MissingBackendError: bcrypt: no backends available -- recommend you install one (e.g. 'pip install bcrypt')

Reply:
worked after installing bcrypt with pip

Reply:
Added to bootstrap